samples/csharp_dotnetcore/49.qnamaker-all-features/Dialog/QnAMakerBaseDialog.cs

```diff
@@ -55,16 +57,16 @@ protected override Task<QnAMakerOptions> GetQnAMakerOptionsAsync(DialogContext d
 
         protected async override Task<QnADialogResponseOptions> GetQnAResponseOptionsAsync(DialogContext dc)
         {
-            var noAnswer = (Activity)Activity.CreateMessageActivity();
-            noAnswer.Text = this._configuration["DefaultAnswer"] ?? DefaultNoAnswer;
+            var defaultAnswerActivity = (Activity)Activity.CreateMessageActivity();            
```

Comment:
The pattern used in the samples is to use MessageFactory.Text() which would alleviate the need to cast and provide a one-line method of setting the Text on the Activity.

```cs
var defaultAnswerActivity = MessageFactory.Text(this.DefaultAnswer);
```

https://github.com/microsoft/BotBuilder-Samples/search?l=C%23&q=MessageFactory.Text

Reply:
changed as suggested